web_relational_enhancer/README.rst

```diff
@@ -0,0 +1,99 @@
+.. image:: https://img.shields.io/badge/License-LGPL%20v3-blue.png
+   :target: https://www.gnu.org/licenses/lgpl-3.0
+   :alt: License: LGPL-3
+
+
+=======================
+Web Relational Enhancer
+=======================
+
+
+This module allow to add information to the 'display_name' on
```

Comment:
hmm is this whole machinery really required just to change the display name?
I mean, you can use `context="{'use_special_readme': True}"` on the field declaration and customize display name/ name_get to achieve the same result.

Can you elaborate on the use case and the benefit of maintaining this big piece of JS? Tnx :)

Reply:
Yes, because it changes the display name only on the dropdown widget, leaving the return values of those methods untouched.

Reply:
this is exactly what I'm talking about :)

you can set the ctx on the field itself and this will enable name_get override (which is used to show the label on the dropdown) only on this field an only when you set the ctx key...

I'm still looking for an answer then :)

Reply:
Yeah, ok. I didn't understand what you meant on the first answer. I thought it was a known feature instead of a keyword which you check on the name_get override. 

Your solution is better on a dynamic use case, but I still think it could be good also to have a static way to do that. It is simpler and faster from the usage point of view, because you only have to specify the fields, you want to see in the label, inside the view field options.

Reply:
I still don't agree... you can achieve what you want like:

```xml
context="{'display_fields': ['a', 'b','c']}"
```
then in this module you can simply have the name_get override and use those fields in sequence.
Or  you can even support a format pattern:
```xml
context="{'display_fields': ['a', 'b','c'], 'display_name_pattern': '{c}: {a} - {b}'}"
```
and you are done. This is the only thing you have to do as a user to enjoy the feature.

Just provide the base logic in this module (or a new one, like "web_display_name_improved" or similar...) to handle this context data.
You'll have a few python lines to maintain instead of this huge JS.

Reply:
You're right. I already used it, I don't know why I did not think about it this time. I'll do it asap.

Thanks for the support.